admin_manual/maintenance/backup.rst

```diff
@@ -48,6 +48,10 @@ database engine. To backup MySQL/MariaDB::
 
     mysqldump --single-transaction -h [server] -u [username] -p[password] [db_name] > nextcloud-sqlbkp_`date +"%Y%m%d"`.bak
 
+If you use `4 byte support <https://docs.nextcloud.com/server/latest/go.php?to=mysql_4byte_support`_ in MySQL/MariaDB (needed for emoji), you will need to add ``--default-character-set=utf8mb4`` like this::
```

Comment:
nöd

Reply:
Unknown target name: "4 byte support <https://docs.nextcloud.com/server/latest/go.php?to=mysql_4byte_support".

Reply:
what does nod mean?